src/ER/Ringdowns.js

```diff
@@ -1,20 +1,54 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
+import ApiService from '../ApiService';
 
 import Heading from '../Components/Heading';
 import RingdownCard from '../Components/RingdownCard';
 import Ringdown from '../Models/Ringdown';
 
 function Ringdowns({ ringdowns }) {
+  const [enroute, setEnroute] = useState([]);
+
   const waiting = ringdowns.filter(
     (r) => r.currentDeliveryStatus === Ringdown.Status.ARRIVED || r.currentDeliveryStatus === Ringdown.Status.OFFLOADED
   );
-  const enroute = ringdowns.filter(
-    (r) =>
-      r.currentDeliveryStatus !== Ringdown.Status.ARRIVED &&
-      r.currentDeliveryStatus !== Ringdown.Status.OFFLOADED &&
-      r.currentDeliveryStatus !== Ringdown.Status.RETURNED_TO_SERVICE
-  );
+
+  useEffect(() => {
+    // set enroute ringdowns once data comes back from server.
+    const handler = setTimeout(() => {
+      setEnroute(
+        ringdowns.filter(
+          (r) =>
+            r.currentDeliveryStatus !== Ringdown.Status.ARRIVED &&
+            r.currentDeliveryStatus !== Ringdown.Status.OFFLOADED &&
+            r.currentDeliveryStatus !== Ringdown.Status.CANCEL_ACKNOWLEDGED &&
+            r.currentDeliveryStatus !== Ringdown.Status.REDIRECT_ACKNOWLEDGED &&
+            r.currentDeliveryStatus !== Ringdown.Status.RETURNED_TO_SERVICE
+        )
+      );
+    }, 100);
+    return () => {
+      clearTimeout(handler);
+    };
+  });
```

Comment:
I'm not sure why this needs to be in a useEffect... does it really not work as it was before?